src/hooks/useUser.js

```diff
@@ -1,15 +1,15 @@
 import { useMemo, useContext } from 'react'
 import IsPublicContext from 'components/IsPublicContext'
 import { getShortNameFromClient, getUserNameFromUrl } from 'lib/utils'
+import { useClient } from 'cozy-client'
 
-function useUser({ userName: providedUserName, cozyClient }) {
+function useUser() {
+  const client = useClient()
   const isPublic = useContext(IsPublicContext)
   const userName = useMemo(
     () =>
-      providedUserName ||
-      getUserNameFromUrl() ||
-      (isPublic ? '?' : getShortNameFromClient(cozyClient)),
-    [cozyClient, providedUserName, isPublic]
+      getUserNameFromUrl() || (isPublic ? '?' : getShortNameFromClient(client)),
+    [client, isPublic]
   )
   const userId = userName
   return { userId, userName }
```

Comment:
(nit: since this code rely on hook & context so React, should be `jsx`. To me `.js` should be framework agnostic)

Reply:
I renamed all the hooks that had React to jsx